security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The {cert-manager-operator} supports using ACME CA servers, such as Let's Encrypt, to issue certificates.
+The {cert-manager-operator} supports using ACME CA servers, such as Let's Encrypt, to issue certificates. Explicit credentials are configured by specifying the secret details in the `Issuer` API object. Ambient credentials are extracted from the environment, metadata services, or local files which are not explicitly configured in the `Issuer` API object.
 
 // About ACME issuers
 include::modules/cert-manager-acme-about.adoc[leveloffset=+1]
```

Comment:
Under “Supported DNS-01 providers”, can you please remove the note that says: 
```
The cert-manager Operator for Red Hat OpenShift does not support using Google workload identity federation.
```
As we will be supporting GCP with Workload Identity moving forward.